Simplenote/src/main/java/com/automattic/simplenote/utils/BaseExt.kt

```diff
@@ -0,0 +1,21 @@
+package com.automattic.simplenote.utils
+
+import android.widget.Toast
+import androidx.annotation.ColorRes
+import androidx.annotation.StringRes
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+
+fun AppCompatActivity.toast(@StringRes resId: Int, length: Int = Toast.LENGTH_SHORT) {
+    Toast.makeText(this, getString(resId), length).show()
+}
+
+fun AppCompatActivity.getColorStr(@ColorRes color: Int): String {
+    val hintColor = ContextCompat.getColor(this, color)
+    val hintHexColor = hintColor.toHexString().replace("ff", "")
+    return "#$hintHexColor"
+}
+
+fun Int.toHexString(): String {
+    return Integer.toHexString(this)
+}
```

Comment:
Minor (🔍): Consider adding a new line at the end of this file. Actually, GitHub also warns you about it, nice! 😄 

PS: Same apply to all other files. Actually, consider adding that setting to your AS so that it is done for you automatically from now on.

Reply:
🤯 , did not know about this. Thanks! Fixed in 55059c868d009e4e830bd484b0b2ae4d2e091c7a.